src/header/js/header-menus.js

```diff
@@ -230,27 +240,23 @@ export default ({variant = 'desktop'}) => {
 	function renderSingle({hasMenuItems, $subcontent, item, uuid, suuid}) {
 		let columns = '';
 
-		if (item.menus.length >= 10) {
-			if (item.menus.length % 2 === 0) {
-				const multi = item.menus.length / 2;
-				
-				columns = $('div', {class: `${prefix}-sublist--col-wrapper`},
-					createMenuColumns(item.menus.slice(0, multi)),
-					createMenuColumns(item.menus.slice(parseInt(multi), item.menus.length - 1))
-				);
-			} else if (item.menus.length % 3 === 0) {
-				const multi = item.menus.length / 3;
-
-				columns = $('div', {class: `${prefix}-sublist--col-wrapper`},
-					createMenuColumns(item.menus.slice(0, multi)),
-					createMenuColumns(item.menus.slice(parseInt(multi), parseInt(multi * 2))),
-					createMenuColumns(item.menus.slice(parseInt(multi * 2), item.menus.length - 1))
-				);
-			}
+		if (item.menus.length >= 10 && item.menus.length <= 18) {
+			const multi = Math.ceil(item.menus.length / 2);
+			columns = $('div', {class: `${prefix}-sublist--col-wrapper columns2`},
+				createMenuColumns(item.menus.slice(0, multi)),
+				createMenuColumns(item.menus.slice(multi, item.menus.length))
+			);
+		} else if (item.menus.length > 18 && item.menus.length <= 27) {
+			const multi = Math.ceil(item.menus.length / 3);
+			columns = $('div', {class: `${prefix}-sublist--col-wrapper columns3`},
```

Comment:
using the class `columns3` is kind of risky as it's so generic. Would be better to do something like `${prefix}-columns-3` to namespace it.

Reply:
@paulcpederson made those updates.. Good call!